ebs/ebs.go

```diff
@@ -457,6 +458,26 @@ func (d *Driver) MountVolume(req Request) (string, error) {
 		return "", err
 	}
 
```

Comment:
Any reason to move the formatting logic to Mount? This workflow is already established in `CreateVolume()` at the where attachment happens. Look at [line 407](https://github.com/medallia/convoy/pull/7/files#diff-aa4f7661033015841b27167793bcd14fR407) in this file; although today it only happens when a new volume is created. To this you can add another check, , that will take the volume ID and do a query to AWS. 
```
if format || volumeNeedsFS(volume.EBSID) {
...
d.ebsService.DeleteTags(volume.EBSID, needFsTag)
}
```